roles/st2/tasks/main.yml

```diff
@@ -1,17 +1,25 @@
 ---
-- name: Install latest st2 package
+- name: Install libffi-devel on EL6
   become: yes
-  apt:
+  package:
+    name: ftp://fr2.rpmfind.net/linux/centos/6/os/x86_64/Packages/libffi-devel-3.0.5-3.2.el6.x86_64.rpm
+    state: latest
+  when: ansible_os_family == "RedHat" and ansible_distribution_major_version == "6"
+  ignore_errors: yes  # Ignore error when already installed
+
+- name: Install latest st2 package (stable)
+  become: yes
+  package:
     name: st2
     state: latest
   when: st2_version == "latest"
   notify:
     - restart st2
   tags: skip_ansible_lint
 
-- name: Install latest st2 package
+- name: Install latest st2 package (not stable)
```

Comment:
Same as https://github.com/StackStorm/ansible-st2/pull/66/files#r97132662

`(not stable)` note could be removed.

We can write `name: Install pinned st2 package version` or similar explanation instead.